Directory.Build.props

```diff
@@ -47,6 +47,9 @@
     <WarningsNotAsErrors>$(WarningsNotAsErrors);CS1591</WarningsNotAsErrors>
     <!-- xUnit1004 = warns about skipped tests. Make this a non-fatal build warning. -->
     <WarningsNotAsErrors>$(WarningsNotAsErrors);xUnit1004</WarningsNotAsErrors>
+    <!-- The assembly 'contentFiles\any\any\x64\aspnetcorev2.dll' is not inside the 'lib' folder and hence it won't be added as a reference when the package is installed into a project.
```

Comment:
So, the warning will still appear, right? I think a better fix would be to suppress completely. This can be done by doing something like this in Microsoft.AspNetCore.Server.IntegrationTesting.IIS.csproj :

```xml
<!-- 
 The assembly 'contentFiles\any\any\x64\aspnetcorev2.dll' is intentionally
 not inside the 'lib' folder and hence it won't be added as a reference when
 the package is installed into a project. This suppresses the NuGet warning
 about this structure 
-->
<NoWarn>$(NoWarn);NU5100</NoWarn>
```